diffpy/nmf_mapping/nmf_mapping/nmf_mapping_code.py

```diff
@@ -330,7 +332,7 @@ def reconstruction_error_plot(df_reconstruction_error, show=True):
     """
     try:
         plt.style.use(bg_mpl_style)
-    except:
+    except Exception:
```

Comment:
see above.  Also, why is this happening multiple times.  Don't we want to just set this at the top of the file and be done with it?